pep/app/pods/most-cited/template.hbs

```diff
@@ -9,12 +9,12 @@
                 }}
             </h4>
             <div class="d-flex align-items-center">
-                <Button
+                <Button::WithTooltip
                     @type="link"
                     @icon="download"
                     @iconSize="lg"
                     @action={{this.downloadCSV}}
-                    title={{t "common.export"}}
```

Comment:
How does this work with accessibility? Do you still need title as well?

Reply:
The addon actually smartly adds an `aria-describedby` attribute on the anchor element (the `<button>` here) that points to the id of the tooltip content container, so we should be good accesibility-wise I think.